style.css

```diff
@@ -0,0 +1,718 @@
+  font-size: 12px;
+  letter-spacing: 1px;
+  flex-shrink: 1;
+}
+
+.item:hover {
+  outline: 2px solid #d3d3d3;
+}
+
+.item-icon {
+  padding-right: 4%;
+  margin: 15px 0;
+}
+
+.main-program {
+  font-family: "Cocogoose", sans-serif;
+  color: #d3d3d3;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.program-title-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.program-subtitle {
+  font-size: 1rem;
+  font-family: "Cocogoose", sans-serif;
+  color: #ec5242;
+}
+
+.hr-title {
+  border: 1px solid #ec5242;
+  width: 20%;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  margin: 40px 0 20px 0;
+}
+
+.program-button {
+  padding: 5% 7%;
+  font-family: "lato", sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  margin: 0 auto;
+  background-color: #ec5242;
+  color: #f0f2f5;
+  border: 0;
+}
+
+.see-more {
+  display: none;
+}
+
+.featured-specialists {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+.specialist-heading {
+  font-size: 24px;
+  color: #272a31;
+  font-family: "Cocogoose", sans-serif;
+}
+
+.featured-specialists-container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 3%;
+}
+
+.specialist-content {
+  display: flex;
+  gap: 5%;
+  padding: 1.5rem;
+  background-image: url("./images/chess.svg");
+  background-repeat: no-repeat;
+  background-size: 70px;
+  background-position: left top;
+}
+
+.specialist-image {
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: 55%;
+  margin-top: 30px;
+  width: 115px;
+  height: 115px;
+  margin-right: 2%;
+}
+
+.specialist-name {
+  font-size: 20px;
+  text-align: left;
+  font-weight: 900;
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+  margin-bottom: 6px;
+}
+
+.specialist-title {
+  font-size: 13px;
+  text-align: left;
+  color: #ec5242;
+  font-style: italic;
+  font-weight: 900;
+  font-family: "Lato", sans-serif;
+}
+
+.sponsors {
+  background-color: #272a31;
+  color: #d3d3d3;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.sponsor-heading {
+  font-size: 40px;
+  font-family: "Lato", sans-serif;
+}
+
+.sponsor-items {
+  list-style: none;
+  display: flex;
+  width: 100%;
+  font-size: 20px;
+  padding-left: 0;
+}
+
+.project-rights-home {
+  display: flex;
+  align-items: center;
+  padding: 4%;
+}
+
+.footer-logo {
+  width: 30%;
+  height: 50%;
+  margin-right: 10%;
+}
+
+/* About section */
+
+.about-introduction {
+  display: flex;
+  padding-top: 20%;
+  color: #ec5242;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-introduction-message {
+  font-size: 40px;
+  font-weight: 500;
+  width: 100%;
+  text-align: center;
+  font-family: "Cocogoose", sans-serif;
+}
+
+.about-description {
+  width: 100%;
+  background-color: #e7e7e7;
+  border: #fff solid 2px;
+  color: #58595a;
+  padding: 0.8rem;
+  text-align: center;
+}
```

Comment:
- [ ] Let's fix the background color on the about page description currently it is more gray than white, :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217489744-ad681a05-d707-41d1-a935-9927aa9460b9.png)
how about `background-color: #f9f9f9`